app/src/docker/docker-compose.yml

```diff
@@ -356,6 +356,8 @@ services:
     environment:
       <<: *common-vars
       <<: *rabbitmq-placeholder-vars
+      # A mock-bgs-api needs to be created and exposes an unused port
+      BGS_BASE_URL: http://mock-bgs-api:9020
```

Comment:
Expecting @greene858 to provide the mock-bgs-api container. Note other mock containers also expose ports, so don't collide with them.

Reply:
Draft PR for mock-bgs-api is open: https://github.com/department-of-veterans-affairs/abd-vro/pull/1420